src/plugins/omemo/Plugin.ts

```diff
@@ -100,6 +103,42 @@ export default class OMEMOPlugin extends EncryptionPlugin {
       });
    }
 
+   private getDeviceListFromContact(contact: IContact)
+   {
+      let connection = this.pluginAPI.getConnection();
+      let _self = this;
+      return connection.getPEPService().retrieveItems('eu.siacs.conversations.axolotl.devicelist',contact.getJid().bare).then(function(data) {
+
+            return new Promise(function(resolve, reject) {
```

Comment:
No need for `_self` with arrow functions.

Reply:
changed to lamda notation and removed _self var